### PR TITLE
Fix golint warning in labels.go

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@cn.fujitsu.com>

### CODE REVIEW

File: pkg/labels/labels.go [hunk 1]
```diff
@@ -301,16 +301,14 @@ func Compare(a, b Labels) int {
 		if a[i].Name != b[i].Name {
 			if a[i].Name < b[i].Name {
 				return -1
-			} else {
-				return 1
 			}
+			return 1
```
Comment (roidelapluie) [inline pkg/labels/labels.go]: This is by design as it is easier to reason about.

File: pkg/labels/labels.go [hunk 2]
```diff
@@ -301,16 +301,14 @@ func Compare(a, b Labels) int {
...
 		}
 		if a[i].Value != b[i].Value {
 			if a[i].Value < b[i].Value {
 				return -1
-			} else {
-				return 1
 			}
+			return 1
```
Comment (roidelapluie) [inline pkg/labels/labels.go]: This is by design as it is easier to reason about.

**Comment by roidelapluie** [comment]
Thank you for your pull request. Can you remove the 'if' as it was left as it by design?

**Comment by fenggw-fnst** [comment]
Thanks for your review.
Done, PTAL, thanks again.

**Comment by roidelapluie** [comment]
Thanks!!

**Pull request merged**